cmd/experimental/kjobctl/pkg/cmd/create/create_test.go

```diff
@@ -1131,6 +1131,72 @@ error_path=$(unmask_filename "$SBATCH_ERROR")
 			},
 			wantOutPattern: `job\.batch\/.+ created\\nconfigmap\/.+ created`,
 		},
+		"should create slurm with --mem flag": {
```

Comment:
Could you add a test case for --mem=1G, to see how the memory will be formatted after dividing (as we need to use another unit)

Reply:
Done.